### PR TITLE
cogapp: initial add

### PR DESCRIPTION
**Pull request by johannesring**

### CODE REVIEW

File: recipes/cogapp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% set version = "2.5" %}
+{% set sha256 = "b0c8efdc7d232b9e66064bae93c259946fdabad153a8c78025bc173f7245bfd7" %}
+
+package:
+  name: cogapp
+  version: {{ version }}
+
+source:
+  fn: cogapp-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/83/20/6cf41fe07aaa3f803988598326c7e31290544b2fb53d6f8868756c5b663f/cogapp-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/cogapp/meta.yaml]: Please use the same URL structure as in the `example`.

Comment (johannesring) [inline recipes/cogapp/meta.yaml]: Thanks, this is fixed now.

File: recipes/cogapp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - cogapp
+
+about:
+  home: http://nedbatchelder.com/code/cog
+  license: MIT
```
Comment (jakirkham) [inline recipes/cogapp/meta.yaml]: Could we please raise an issue upstream to get a license file and have it added to `MANIFEST.in`? Please note to upstream that we are violating the terms of the MIT license by not including it.

**Comment by edisongustavo** [comment]
Hi, I'm interested in this PR too.

@jakirkham I've sent an e-mail to the author (ned@nedbatchelder.com) with the contents:

```
Hi,

I want to add cogapp to conda-forge, however, one of the issues raised is that the sources don't have a LICENSE file, which is necessary to conform to the MIT license.

The pull-request is here: https://github.com/conda-forge/staged-recipes/pull/1289
Notice that I'm not the original author of this PR.

Would it be a problem for you to add this file and do a minor release with it?

Thanks,
Edison Gustavo Muenz
```

So now we're waiting.

**Comment by edisongustavo** [comment]
@johannesring would you kindly add me as a maintainer too?

**Comment by johannesring** [comment]
@edisongustavo - sure, I have added you now.

**Pull request merged**